test/prefer-string-slice.js

```diff
@@ -32,6 +35,46 @@ test({
 			output: 'foo.slice()',
 			errors: errorsSubstr
 		},
+		{
+			code: 'foo?.substr()',
+			output: 'foo?.slice()',
+			errors: errorsSubstr
+		},
+		{
+			code: 'foo.bar?.substring()',
+			output: 'foo.bar?.slice()',
+			errors: errorsSubstring
+		},
+		{
+			code: 'foo?.[0]?.substring()',
+			output: 'foo?.[0]?.slice()',
+			errors: errorsSubstring
+		},
+		{
+			code: 'foo.bar.substr?.()',
+			output: 'foo.bar.slice?.()',
```

Comment:
I'm not sure about this, the old one is testing object has `.substr`, but new one testing `.slice` exists, if someone mean to test it is a `string` and `foo.bar` is an `array`, it's broken.

Reply:
Hope no one use this...

Reply:
As you want